package.json

```diff
@@ -7,14 +7,18 @@
     "build": "gatsby build",
     "develop": "gatsby develop",
     "format": "prettier --write \"**/*.{js,jsx,ts,tsx,json,md}\"",
-    "start": "npm run develop",
+    "start": "export SET NODE_OPTIONS=--openssl-legacy-provider && npm run develop",
```

Comment:
Why is this needed? Doesn't it work with older node versions?

I'm guessing node 18 isn't supported, but that's because Gatsby is outdated (we can update it as well later).

Reply:
I don't remember, but it seems like it's not needed